### PR TITLE
Add new keyboard "miniZone"

### PR DESCRIPTION
**Pull request by takashicompany**
## Description

Thank you for your assistance.
I have created a new keyboard and would like to have it added to QMK Firmware.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/takashicompany/minizone/info.json [hunk 1]
```diff
@@ -0,0 +1,239 @@
+{
```
Comment (fauxpark) [inline keyboards/takashicompany/minizone/info.json:1]: Please fix the positioning and ordering:
image
image

File: keyboards/takashicompany/minizone/config.h [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
```
Comment (fauxpark) [inline keyboards/takashicompany/minizone/config.h:135]:
```suggestion
```

File: keyboards/takashicompany/minizone/minizone.h [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    k21, k22, k23, k24, k25, k26, k27, k28, k29, k30, \
+         k31, k32, k33, k34, k35, k36, k37, k38 \
+) { \
+    { k01, k02, k03, k04, k05, k06, k07 }, \
+    { k08, k09, k10, k11, k12, k13, k14 }, \
+    { k15, k16, k17, k18, k19, k20, k21 }, \
+    { k22, k23, k24, k25, k26, k27, k28 }, \
+    { k29, k30, k31, k32, k33, k34, k35 }, \
+    { k36, k37, k38, KC_NO, KC_NO, KC_NO, KC_NO } \
+}
```
Comment (fauxpark) [inline keyboards/takashicompany/minizone/minizone.h:28]:
```suggestion
#define LAYOUT( \
    k00, k01, k02, k03, k04, k05, k06, k10, k11, k12, \
    k13, k14, k15, k16, k20, k21, k22, k23, k24, k25, \
    k26, k30, k31, k32, k33, k34, k35, k36, k40, k41, \
         k42, k43, k44, k45, k46, k50, k51, k52 \
) { \
    { k00, k01, k02, k03, k04, k05, k06 }, \
    { k10, k11, k12, k13, k14, k15, k16 }, \
    { k20, k21, k22, k23, k24, k25, k26 }, \
    { k30, k31, k32, k33, k34, k35, k36 }, \
    { k40, k41, k42, k43, k44, k45, k46 }, \
    { k50, k51, k52, XXX, XXX, XXX, XXX } \
}
```

File: keyboards/takashicompany/minizone/minizone.h [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
```
Comment (fauxpark) [inline keyboards/takashicompany/minizone/minizone.h:6]:
```suggestion
#include "quantum.h"

#define XXX KC_NO
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

File: keyboards/takashicompany/minizone/config.h [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/takashicompany/minizone/config.h:70]: Could you explicitly define which animations are enabled? 

```suggestion
#    define RGBLIGHT_EFFECT_BREATHING
#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#    define RGBLIGHT_EFFECT_SNAKE
#    define RGBLIGHT_EFFECT_KNIGHT
#    define RGBLIGHT_EFFECT_CHRISTMAS
#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
//#    define RGBLIGHT_EFFECT_RGB_TEST
//#    define RGBLIGHT_EFFECT_ALTERNATING
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/takashicompany/minizone/readme.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# miniZone
+
+![miniZone](https://github.com/takashicompany/minizone/blob/master/images/qmk.jpg?raw=true)
```
Comment (drashna) [inline keyboards/takashicompany/minizone/readme.md:3]: Could you rehost this on an image hosting site, such as imgur?

Comment (takashicompany) [reply keyboards/takashicompany/minizone/readme.md:3]: @drashna 

Thanks for the review.
In a previous Pull Request for another keyboard, the review passed with no problem with an image link on the Github
If there are no problems, we would like to continue as is.

Comment (takashicompany) [reply keyboards/takashicompany/minizone/readme.md]: Yes, sir.
Thank you for your kind explanation.
URL has been changed.

**Review by drashna — APPROVED** [review]

**Comment by takashicompany** [comment]
@fauxpark 
I am sorry to bother you, but I would appreciate your review.

**Pull request merged**